File: app/src/main/java/com/example/toolgather/Adapter/adapter_hemo_v1/SectionsPagerAdapter.java
package com.example.toolgather.Adapter.adapter_hemo_v1;


import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import java.util.List;

/**
 * @author Administrator : HYY
 * TIME    2019/11/4
 * 备注 :  首页Adapter
 */
public class SectionsPagerAdapter extends FragmentPagerAdapter {
    List<Fragment> fragment;

    public SectionsPagerAdapter(FragmentManager fm, List<Fragment> fragments) {
        super(fm);
        this.fragment = fragments;
    }

    @Override
    public Fragment getItem(int position) {
        return fragment.get(position);
    }

    @Override
    public int getCount() {
        return fragment.size();
    }
}

File: app/src/main/java/com/hyy/hytool/adapter/SectionsPagerAdapter.kt
package com.hyy.hytool.adapter

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

/**
 * @Author : HYY
 * 描述:      首页Adapter
 * 时间:      2022/05/2022/5/31
 * 软件包名:   com.hyy.hytool.adapter
 */
class SectionsPagerAdapter (fm: FragmentManager?, var fragment: List<Fragment>) : FragmentPagerAdapter(
    fm!!
) {
    override fun getItem(position: Int): Fragment {
        return fragment[position]
    }

    override fun getCount(): Int {
        return fragment.size
    }
}